[DOCS] Updates Watcher docs to match updated UI (#40106)
elastic/kibana: docs/management/watcher-ui/index.asciidoc

[role="xpack"]
[[watcher-ui]]
== Watcher

> Watcher is an {es} feature that you can use to create actions based on 

Reviewer: I think in general we're not using "Watcher" as much anymore. If you look on https://www.elastic.co/subscriptions and https://www.elastic.co/products/stack/alerting it's not mentioned there. 

I recommend using the alert-features attribute from https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc

For example:
```suggestion
The {alert-features} enable you to create actions based on 
```

conditions, which are periodically evaluated using queries on your data. 
Watches are helpful for analyzing mission-critical and business-critical 
streaming data. For example, you might watch application logs for performance 
outages or audit access logs for security threats.

With the Watcher UI, you can:

* Create a simple threshold watch
* View your watch history and action status
* Acknowledge a watch to prevent too many executions
* Edit, deactivate, and delete a watch
* Create more advanced watches using API syntax

[role="screenshot"]
image:management/watcher-ui/images/watches.png["Watcher list"]

{stack-ov}/xpack-alerting.html[Alerting on cluster and index events]
is a good source for detailed 
information on how watches work. If you are using the UI to create a 
threshold watch, take a look at the different watcher actions. If you are 
creating an advanced watch, you should be familiar with the parts of a 
watch&#8212;input, schedule, condition, and actions.

[float]
[[watcher-security]]
=== Required roles for Watcher

> To use Watcher, you must have the built-in `kibana_user` role.  

Reviewer: It seems to me that this section only applies when security features are enabled, so it's a good idea to add that qualification. For example:

```suggestion
If the {es} {security-features} are enabled, to use {alert-features} in {kib}, you must have the built-in `kibana_user` role.  
```

> You must also have one of the following Watcher roles:

Reviewer: ```suggestion
You must also have one of the following built-in roles:
```

* `watcher_admin` can perform all Watcher actions, such as creating a new watch 
or editing an existing one.

* `watcher_user` can view all existing watches, but cannot create or edit watches.

You can manage roles in *Management > Security > Roles*, or use the 
<<role-management-api, Kibana Role Management API>>. Watches are shared between 

> all users with the same role.  

Reviewer: When we're describing security requirements, we often start by saying which privileges they require (e.g. manage_watcher or monitor_watcher cluster privileges per https://www.elastic.co/guide/en/elastic-stack-overview/master/security-privileges.html . Then we mention the built-in roles in case they don't want to add these privileges to their existing roles.

[role="xpack"]
[[watcher-ui]]
== Watcher

Watcher is an {es} feature that you can use to create actions based on 
conditions, which are periodically evaluated using queries on your data. 
Watches are helpful for analyzing mission-critical and business-critical 
streaming data. For example, you might watch application logs for performance 
outages or audit access logs for security threats.

With the Watcher UI, you can:

* Create a simple threshold watch
* View your watch history and action status
* Edit, deactivate, and delete a watch
* Create more advanced watches using API syntax

[role="screenshot"]
image:management/watcher-ui/images/watches.png["Watcher list"]

{stack-ov}/xpack-alerting.html[Alerting on cluster and index events]
is a good source for detailed 
information on how watches work. If you are using the UI to create a 
threshold watch, take a look at the different watcher actions. If you are 
creating an advanced watch, you should be familiar with the parts of a 
watch&#8212;input, schedule, condition, and actions.

[float]
[[watcher-security]]
=== Watcher security

If the {es} {security-features} are enabled, you must have the

> {stack-ov}/security-privileges.html[manage-watcher or monitor_watcher]

Reviewer: ```suggestion
{stack-ov}/security-privileges.html[`manage_watcher` or `monitor_watcher`]
```
